systers_portal/templates/community/community_search.html

```diff
@@ -0,0 +1,7 @@
+{%extends 'base.html'%}
+{%block content%}
```

Comment:
nitpick: could you improve the formatting on this?

```suggestion
{% block content %}
```